R/ioslides_presentation.R

```diff
@@ -1,7 +1,266 @@
-
-
-
-
+#' Convert to an ioslides Presentation
+#'
+#' Format for converting from R Markdown to an
+#' \href{https://code.google.com/p/io-2012-slides/}{ioslides} presentation.
+#'
+#' @param logo Path to file that includes a logo for use in the presentation
+#'   (should be square and at least 128x128).
+#' @param slide_level Header level to consider as slide separator (Defaults to
+#'   header 2).
+#' @param incremental \code{TRUE} to render slide bullets incrementally.
+#'   Note that if you want to reverse the default incremental behavior for an
+#'   individual bullet you can preceded it with \code{>}.
+#'   For example: \emph{\code{> - Bullet Text}}.
+#' @param fig_width Default width (in inches) for figures.
```

Comment:
I think it will be nice to inherit the docs from other output formats like `@inhertParams html_document`, so we don't have to repeat the documentation of these arguments.

Reply:
This definitely showed where the `ioslides_presentation` docs hadn't kept up with `html_document`.